src/app/components/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './SearchField.module.css';
+import ButtonLarge from '../ButtonLarge/ButtonLarge';
+
+type SearchFieldProps = {
+  type: 'Search' | 'Add';
+  locationSearchValue: string;
+  setLocationSearchValue: (locationSearchValue: string) => void;
+  handleSubmit: (event: React.FormEvent) => void;
+};
+
+export default function SearchField({
+  type,
+  locationSearchValue,
+  setLocationSearchValue,
+  handleSubmit,
+}: SearchFieldProps): JSX.Element {
+  return (
+    <form className={styles.formSection} onSubmit={handleSubmit}>
+      <label>
+        <input
+          type="text"
+          value={locationSearchValue}
+          className={styles.formInput}
+          placeholder={
+            type === 'Search' ? 'Search' : type === 'Add' ? 'Add' : `${type}`
+          }
```

Comment:
Would it be enough here to make this:
```suggestion
          placeholder={type}
```
?